README.md

```diff
@@ -1,11 +1,8 @@
 # Poetry export plugin
 
-This package is a plugin that allows the export of locked packages to various formats.
-
-**Note**: For now, only the `requirements.txt` format is available.
-
-This plugin provides the same features as the existing `export` command of Poetry which it will eventually replace.
-
+This package is a fork of [poetry/poetry-export-plugin](https://github.com/tailify/poetry-export-plugin) plugin
```

Comment:
issue: If this is getting proposed to be merged into the main repo, it doesn't make sense to reference your fork.